src/main.py

```
@@ -34,68 +36,75 @@
             for url, error in invalid_feeds.items():
                 table.add_row(url, error)
             console.print(table)
-            console.print(f"\nInvalid feeds have been saved to '{manager.invalid_file}'")
+            console.print(
+                f"\nInvalid feeds have been saved to '{manager.invalid_file}'"
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)